av1-rtp-spec.md

```diff
@@ -807,6 +812,9 @@ To facilitate the work of selectively forwarding portions of a scalable video bi
 
 To reduce overhead, repetitive information can be predefined with templates and sent once. Subsequent packets refer to a template containing predefined information. In particular, when a video encoder uses an unchanging (static) prediction structure to encode a scalable bitstream, parameter values used to describe the bitstream repeat in a predictable way. The techniques described in this document provide means to send repeating information as predefined templates that can be referenced at future points of the bitstream. Since a reference index to a template requires fewer bits to convey than the associated structures themselves, header overhead can be substantially reduced.
 
+**Note:** Templates may be sent multiple times, in order to ensure that the templates are received.

```

Comment:
This notes looks out of topic, and, I think it adds confusion rather than clarification:

Presence of templates is strictly bound to the beginning of a coded video sequence:
"template_dependency_structure_present_flag MUST be set to 1 for the first packet of a coded video sequence, and MUST be set to 0 otherwise."

It is true templates are sent multiple times sometimes. In simulcast case they MUST be sent multiple times (once per independent layer), in other cases they MUST be sent once. There is no "MAY" in deciding if templates need to be attached.

Reply:
I moved the definition to the introduction (where there was already quite similar text).  That required changing the title of A.6.

I also addressed the two comments above, and changed the "Since" in A.6 back to "In the event of packet loss".